### PR TITLE
[API][Checkout] Implement scenarios for preventing checkout with an empty cart

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/ApiBundle/Resources/config/validation/CompleteOrder.xml [hunk 1]
```diff
@@ -16,20 +16,22 @@
                     xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping http://symfony.com/schema/dic/services/constraint-mapping-1.0.xsd">
 
     <class name="Sylius\Bundle\ApiBundle\Command\Checkout\CompleteOrder">
+        <constraint name="Sylius\Bundle\ApiBundle\Validator\Constraints\OrderNotEmpty">
+            <option name="groups">
+                <value>sylius_checkout_complete</value>
+            </option>
+        </constraint>
         <constraint name="Sylius\Bundle\ApiBundle\Validator\Constraints\OrderProductEligibility">
-            <option name="message">sylius.order.product_eligibility</option>
```
Comment (AdamKasp) [inline src/Sylius/Bundle/ApiBundle/Resources/config/validation/CompleteOrder.xml:20]: Why did you delete it?

**Review by AdamKasp — APPROVED** [review]

**Review by arti0090 — APPROVED** [review]

File: src/Sylius/Bundle/ApiBundle/Resources/config/validation/CompleteOrder.xml (hunk 1 above)
Comment (GSadee) [reply src/Sylius/Bundle/ApiBundle/Resources/config/validation/CompleteOrder.xml:20]: Because the message is already defined in `OrderProductEligibility` constraint

**Comment by AdamKasp** [comment]
Thank you, Grzegorz! :tada:
